parsers/UA.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import arrow
+import dateutil
+import requests
+
+MAP_GENERATION = {
+  'aes': 'nuclear',
+  'tec': 'other', # fossil 
+  'tes': 'coal',
+  'chpp': 'gas',
+  'hpp': 'hydro',
+  'vde': 'wind',
+  'gesgaes' : 'gesgaes', # hydro pumped storage
+  'consumptiongaespump' :'consumptiongaespump' #hydro run of river and poundage
+}
+MAP_OTHER   = {
+  'consumption' : 'consumption',
+  'hour' : 'hour'
+}
+
+tz = 'Europe/Kiev'
+
+def fetch_production(country_code='UA', session=None):
+    r = session or requests.session()
+    
+    today =  arrow.now(tz=tz).format('DD.MM.YYYY')
+    url = 'https://ua.energy/wp-admin/admin-ajax.php'
+    postdata = {
+      'action': 'get_data_oes',
+      'report_date': arrow.now(tz=tz).format('DD.MM.YYYY'),
+      'type': 'day',
+      'rnd': 0.2046847583117225
+    }
+    response = r.post(url, postdata)
+    obj = response.json()
+    allofit = []

```

Comment:
The expected result is an array where each item is a dict with datetime, production etc..
Here you are building an array of array of dict.
I suggest you take a look at how I did with the PE parser, where I also get a series for each of the production source, and need to build a list (array) where each object (dictionary) contains all production modes.